.github/workflows/release-next.yml

```diff
@@ -41,15 +46,48 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+
+      # if the ui_bundle_url is defined download and unpack the dashboard
       -
         name: Download dashboard
+        if: ${{ github.events.inputs.ui_bundle_url != '' }}
         run: |
           mkdir ui
-          wget "${{ env.UI_BUNDLE_URL }}"
+          wget "${{ github.events.inputs.ui_bundle_url }}"
           tar xfz *.tar.gz -C ui
+
+      # otherwise fetch and build the latest dashboard from the repository
+      -
+        name: Checkout Rancher Dashboard UI
+        if: ${{ github.events.inputs.ui_bundle_url == '' }}
+        uses: actions/checkout@v2
+        with:
+          repository: rancher/dashboard
+          ref: epinio-dev
+          submodules: recursive
+          fetch-depth: 0
+      -
+        name: Build Epinio dashboard
+        if: ${{ github.events.inputs.ui_bundle_url == '' }}
+        run: |
+          pushd dashboard
+          ./.github/workflows/scripts/build-dashboard.sh
+          mv $OUTPUT_DIR/$ARTIFACT_NAME ../ui
+          popd
+          rm -rf dashboard
+        env:
+          RANCHER_ENV: epinio
+          EXCLUDES_PKG: rancher-components,harvester
+          EXCLUDES_NUXT_PLUGINS: plugins/plugin,plugins/version
+          OUTPUT_DIR: dist
+          RELEASE_DIR: release
+          ARTIFACT_NAME: rancher-dashboard-epinio-standalone-embed
+
+      #################
+
       -
         name: Run GoReleaser Cross
         run: ./build/bk-release.sh
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          UI_BUNDLE_URL: "${{ github.events.inputs.ui_bundle_url || env.UI_BUNDLE_URL }}"
+          UI_BUNDLE_URL: "${{ github.events.inputs.ui_bundle_url || 'dev' }}"
```

Comment:
not sure if `dev` is valid here (it should be an URL)

Reply:
Yes, that is fine. In the release it's only used to add a label in the docker image, and when we are building it from source then a `dev` value is used. :slightly_smiling_face: